### PR TITLE
add all data store to work dir

### PR DESCRIPTION
**Pull request by upccup**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -86,6 +86,11 @@ func main() {
 			Value: "mixed",
 			Usage: "Server mode, manager|agent|mixed ",
 		},
+		cli.StringFlag{
+			Name:  "work-dir",
+			Value: "./data/",
```
Comment (cmingxu) [inline main.go]: how about move this under tmp directory.

File: main.go [hunk 2]
```diff
@@ -96,7 +101,7 @@ func main() {
 
 		setupLogger(config.LogLevel)
 
-		db, err := bolt.Open(fmt.Sprintf(".bolt.db.%d", config.Raft.RaftId), 0600, nil)
+		db, err := bolt.Open(fmt.Sprintf(c.String("work-dir")+".bolt.db.%d", config.Raft.RaftId), 0600, nil)
```
Comment (cmingxu) [inline main.go]: is it possible move boltdb initialization statement within manager.Start method?

File: main.go (hunk 1 above)
Comment (upccup) [inline main.go]: is not tmp data and in current dir is convenience for us

**Comment by pwzgorilla** [comment]
LGTM

File: main.go (hunk 1 above)
Comment (cmingxu) [inline main.go]: how about make this visible? as hidden directory give me a hint that the data isn't important

**Pull request merged**